src/content/community.md

```diff
@@ -52,6 +53,16 @@ There are several ways to stay up-to-date with Firefox add-on technology and con
 
 Stay up to date with the latest information about add-on technology on the [Add-ons blog](https://blog.mozilla.org/addons).
 
+### Firefox for Android office hours <span id="office-hours"></span>
+
+The Firefox Add-ons team holds one-on-one office hours sessions to help you make your extensions available on Android. This is in addition to the [Firefox for Android Add-ons](https://discourse.mozilla.org/c/add-ons/android/9393) forum, where you can seek advice and assistance anytime.
+
+<details>
+  <summary>Book a session</summary>
+
+  <iframe id="booking-frame" src="https://calendar.google.com/calendar/appointments/schedules/AcZssZ3yNnoZNBdJCTiJjqziXs9GoCYpfW9XZFkB0QWKNy9dfoqqPlCw7UsYh48Ff0i3lZcA5YzygwrU?gv=true" sandbox="allow-same-origin allow-scripts allow-forms allow-popups" style="height: 600px;"></iframe>
```

Comment:
Just wondering - do we need to add `allow-top-navigation` or even `allow-top-navigation-by-user-activation` as well? Only if a navigation triggered from the calendar is an expected part of the functionality.

And maybe also allow-modals if you want to enable things like alert/confirm.

Reply:
The default sandbox value set by the Google Calendar scripts scripts is `"allow-same-origin allow-scripts allow-forms allow-popups allow-popups-to-escape-sandbox allow-storage-access-by-user-activation"`. I tried to trim this list to minimize the capabilities being granted to the sandbox. In retrospect, I _think_ those additional capabilities are reasonable and will therefore go back to the default sandbox settings.

I don't think we need to add navigation-related capabilities since Calendar itself didn't request them.